chain-impl-mockchain/Cargo.toml

```diff
@@ -57,6 +57,7 @@ rand_chacha = "0.3"
 lazy_static = "1.3.0"
 tempfile = "3.1.0"
 rand = "0.8"
+cryptoxide = "0.3"
```

Comment:
I failed to find relevant diff that requires it, but should it not be abstracted anyway in chain-vote / chain-crypto ?

Reply:
Good catch. Removed